src/reviews/views.py

```diff
@@ -132,9 +148,12 @@ def get_context_data(self, **kwargs):
             ),
             'vote': vote_mapping,
             'ordering': self.ordering,
+            'category': self.category,
             'query_string': self.request.GET.urlencode(),
             **self.reviews_state._asdict(),
+            **self.get_category_metrics(context),
         })
+        print(context)
```

Comment:
I guess this is used for debugging? could you help remove it, thanks!

Reply:
Resolved in daa9f1bfcba8ff46b949f461027138e7a37e664d